GitHubCore.ps1

```diff
@@ -735,8 +734,10 @@ function Split-GitHubUri
         repositoryName = [String]::Empty
     }
 
-    if (($Uri -match '^https?://(?:www.)?github.com/([^/]+)/?([^/]+)?(?:/.*)?$') -or
-        ($Uri -match '^https?://api.github.com/repos/([^/]+)/?([^/]+)?(?:/.*)?$'))
+    $gitHubUrlBase = $(Get-GitHubConfiguration -Name "GitHubBaseUrl")
```

Comment:
I'd suggest being consistent with how this is referred to (`UrlBase` vs `BaseUrl`):

```suggestion
    $gitHubBaseUrl = $(Get-GitHubConfiguration -Name "GitHubBaseUrl")
```

Reply:
Resolved with a change to 'hostName' a la API host name.